test.rm

```diff
@@ -0,0 +1,2 @@
+## this is read me file
+# this file is not upload
```

Comment:
- [ ] It seems something is not working right at the moment, my friend. This file as shown below is supposed to be hidden in your `.gitignore` as per [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) but it is currently in your root directory. Kindly add it to your ignore.
- [ ] Also, you would have noticed that the file name is not as required: You should have `test.md` not `test.rm`. Did you see that too :question: 

![image](https://user-images.githubusercontent.com/99715640/218823957-be4eaf5b-933e-4dde-a6af-fdff371e90bc.png)